packages/components/src/border-box-control/border-box-control-linked-button/component.tsx

```diff
@@ -28,10 +28,9 @@ const BorderBoxControlLinkedButton = (
 			<View className={ className }>
 				<Button
 					{ ...buttonProps }
-					variant={ isLinked ? 'primary' : 'secondary' }
 					isSmall
 					icon={ isLinked ? link : linkOff }
-					iconSize={ 16 }
+					iconSize={ 24 }
```

Comment:
In general icons should never be smaller than 24px, with almost no exceptions, because it causes blurry antialiasing of the pixels that were designed for the 24px grid. The only exception I can think of, is the external link icon, which can occasionally be shown in 12px size. But at that size (exact half), the pixels interpolate correctly, so it remains crisp.